model_test.go

```diff
@@ -275,7 +277,9 @@ func TestRBACModelWithCustomData(t *testing.T) {
 
 func TestRBACModelWithPattern(t *testing.T) {
 	e := NewEnforcer("examples/rbac_with_pattern_model.conf", "examples/rbac_with_pattern_policy.csv")
+	e.rm.(*defaultrolemanager.RoleManager).EnablePattern(true)
```

Comment:
We do not need to provide ``EnablePattern()``. When the user adds a function, we can just assume pattern is enabled.